README.md

```diff
@@ -30,15 +30,37 @@ The library support the react native autolink feature.
 
 ```xml
 // file: android/app/src/main/AndroidManifest.xml
-...
-    <uses-permission android:name="android.permission.BLUETOOTH"/>
-    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+<!-- Add xmlns:tools -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="YOUR_PACKAGE_NAME">
+
+    <uses-permission android:name="android.permission.BLUETOOTH" android:maxSdkVersion="30" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" android:maxSdkVersion="30" />
+
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" android:maxSdkVersion="28"/>
+    <uses-permission-sdk-23 android:name="android.permission.ACCESS_FINE_LOCATION" tools:targetApi="Q"/>
```

Comment:
```suggestion
<uses-permission-sdk-23 android:name="android.permission.ACCESS_FINE_LOCATION" tools:targetApi="Q" android:maxSdkVersion="30"/>
```
According to [android docs](https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#assert-never-for-location) I would expect scanner to work without location permission, but in my testing that was not the case (8.1.0 scanning doesn't find anything without location permission).
Is there something that  needs contribution in android module to achieve this, or is my expectation simply wrong for Bluetooth **Low Energy**? 

@marcosinigaglia

Reply:
Inspired by [Davids comment](https://github.com/innoveit/react-native-ble-manager/issues/872#issuecomment-1174574525) I mananged to make it work.
 * Bumped targetSdk and compileSdk from 31 to 32, 
 * added `maxSdkVersion=32` to ACCESS_FINE_LOCATION
 * in js consider location unneeded (like iOS) if `Platform.Version > 30` (not 31)